apps/daimo-web/src/components/PixelBackground.tsx

```diff
@@ -0,0 +1,6929 @@
+export function PixelBackground(
```

Comment:
hmmmmm this will create a huge bundle size, right?

Reply:
recommend linking to asset image

Reply:
I just pulled this out to be a static asset, though it only decreased first load js bundle size by 6kb